backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -134,9 +134,8 @@ def belongs_to(model_name, options = {})
     end
   end
 
-  def resource_not_found
-    flash[:error] = flash_message_for(model_class.new, :not_found)
-    redirect_to collection_url
+  def resource_not_found(*, flash_class: model_class, redirect_url: collection_url)
```

Comment:
I'm okay with `_error = nil` here. I had not thought about the fact that `rescue_from` will pass the error to the method if it takes arguments.

Reply:
Ah, I was unaware that `rescue_from` does that, it was quite confusing until I realized what was happening.
I will replace the 'splat argument' with `_error=nil` then :+1: